net/gatling/patches/series

```diff
@@ -0,0 +1,2 @@
+010-decrease_optimisation.patch
+020-configure_features.patch
```

Comment:
What's for?

Reply:
Thats the file produced by quilt. As far as I got it from the wiki, one should use quilt for having patches in OpenWrt.

Reply:
I understand, but no other packages have it, so feel free to remove it.

Reply:
This file is not necessary.
If you build the package with `make ./packages/feeds/gatling/{clean,compile} V=s QUILT=1` and then create a new patch with qullt in the build directory and then update the patches into the openwrt packages feed with `make ./packages/feeds/gatling/update`, then this file should not be there.

Reply:
Okay, I deleted it from the PR.